src/main/java/org/jabref/logic/ai/AiPreferences.java

```diff
@@ -570,4 +570,25 @@ public String getTemplate(AiTemplate aiTemplate) {
     public StringProperty templateProperty(AiTemplate aiTemplate) {
         return templates.get(aiTemplate);
     }
+
+    /**
+     * Returns whether the selected model is "not safe" to use. By "safe" it is meant that the model is remote model from
+     * reputable companies.
+     * <p>
+     * This function is made for <a href="https://eur-lex.europa.eu/eli/reg/2024/1689/oj">EU act on AI</a>. LLMs are high-risk systems and may generate harmful content.
+     * If user connects to a reputable remote model (OpenAI, Gemini, Mistral AI, etc.), then they are safe to use, as they
+     * are subject to the EU AI act. However, when user selects a local model or models from Hugging Face, then any model
+     * can be used in JabRef, including uncensored and harmful ones.
+     */
+    public boolean isUnsafeModelSelected() {
```

Comment:
Rename to `isSubjectToEuAiAct`?

In all cases do NOT use negations in the method name. Use `isSafeModelSelected`.